### PR TITLE
Update script to use cache_dir

### PR DESCRIPTION
**Pull request by isanghao**

### CODE REVIEW

**Comment by isanghao** [comment]
@eaidova  @zhuo-yoyowz  For GPU, it took quite a long time to load the model. This cache_dir option reduces the model loading time significantly. What about applying this change to all notebooks?

**Comment by zhuo-yoyowz** [comment]
Hi Mingyu,

Thank you for your PR. I think it’s a great idea to use “cache_dir” to reduce the model loading time. I’m not sure if there’s a simple way to apply this change to all notebooks at one time, but I guess we could start use this in notebooks contributed by ourselves first. Maybe @Aidova, ***@***.***> have better suggestions on this. Or we could discuss this in our notebook weekly sync meeting. Thanks~

BR
Zhuo

**Comment by isanghao** [comment]
@eaidova yes, I intentionally committed the empty cache dir in order to avoid adding mkdir code in python.

**Review by zhuo-yoyowz — APPROVED** [review]

**Review by eaidova — APPROVED** [review]

**Comment by isanghao** [comment]
@eaidova  sorry for the belated response. it was added.

File: notebooks/201-vision-monodepth/201-vision-monodepth.ipynb [hunk 1]
```diff
@@ -159,6 +159,7 @@
    "outputs": [],
    "source": [
     "ie = Core()\n",
+    "ie.set_property({'CACHE_DIR': '../cache'});\n",
```
Comment (eaidova) [inline notebooks/201-vision-monodepth/201-vision-monodepth.ipynb:162]:
```suggestion
    "ie.set_property({'CACHE_DIR': '../cache'})\n",
```

**Review by adrianboguszewski — APPROVED** [review]

**Pull request merged**